test/enquirer.js

```diff
@@ -25,33 +23,32 @@ describe('Enquirer', function() {
         message: 'Favorite color?',
         show: false
       })
-      .then(answers => {
-        assert.equal(answers.color, 'orange');
-        cb();
-      });
+        .then(answers => {
+          assert.equal(answers.color, 'orange');
+          cb();
+        });
```

Comment:
I tried to create a formatter configuration for IntelliJ that doesn't change anything in the existing code but failed. 
I'm not sure if it is because you are using another editor (.idea is in the .gitignore :thinking:) with incompatible settings or if there is no strongly imposed formatting.